api/templates/emails/verified_profile_tips.html

```
@@ -0,0 +1,141 @@
+{% extends "emails/base.html" %} 
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)